notes/condition.md

```diff
@@ -53,23 +48,113 @@ where we used $$\|{\bf A}\boldsymbol{x}\| \le \|{\bf A}\| \|\boldsymbol{x}\|, \f
 
 Then
 
-$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\|\Delta \boldsymbol{b}\|}{\|\boldsymbol{b}\|}  \qquad (1)$$
+$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\|\Delta \boldsymbol{b}\|}{\|\boldsymbol{b}\|}  \qquad (1)$$ 
+
+<br>
+
+
+Instead of changing the input $$\boldsymbol{b}$$, we can also add a perturbation to the matrix $$\boldsymbol{A}$$ (input) by a small amount $$\boldsymbol{E}$$, such that 
+
+$$
+\begin{align}
+(\boldsymbol{A} + \boldsymbol{E}) \hat{\boldsymbol{x}} = \boldsymbol{b}
+\end{align}
+$$
+
+and in a similar way obtain:
+
+$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\| \boldsymbol{E}\|}{\|\boldsymbol{A}\|}  $$
 
 
 Therefore, if we know the relative error in input, then we can use the condition number of the system to obtain an upper bound for the relative error of our computed solution (output).
 
+
+#### Examples
+
+PLACEHOLDER
```

Comment:
Fill in the PLACEHOLDER with the solved i-clicker question or something else here.